dspace/config/modules/requestitem.cfg

```diff
@@ -0,0 +1,43 @@
+#---------------------------------------------------------------#
+#----------------REQUEST ITEM CONFIGURATION---------------------#
+#---------------------------------------------------------------#
+
+# Configuration of request-item. Possible values:
+# all - Anonymous users can request an item
+# logged - Login is mandatory to request an item
+# empty/commented out - request-copy not allowed
+#request.item.type = all
+# Should all Request Copy emails go to the helpdesk instead of the item submitter?
+#request.item.helpdesk.override = false
+# Should a rejection of a copy request send an email back to the requester?
+# Defaults to "true", which means a rejection email is sent back.
+# Setting it to "false" results in a silent rejection.
+#request.item.reject.email = true
+# Bundles to use when granting access and inspecting file size, if "all files" is indicated
+# Default: ORIGINAL
+#request.item.grant.bundles = ORIGINAL
+# By default a granted request will send the file as an attachment.
+# If a secure web link to the bitstream(s) should be sent in some cases, enable the following property
+# Default: false
+#request.item.grant.link = true
```

Comment:
I think we should better describe this setting instead of saying "in some cases".  We should say that the link is sent if the files are above a specific size (specified in "filesize") below.